next.config.js

```diff
@@ -15,6 +15,10 @@ const moduleExports = {
   sassOptions: {
     includePaths: [path.join(__dirname, 'src/styles')],
   },
+  typescript: {
+    tsconfigPath: 'tsconfig.build.json',
+  },
+  tsconfigPath: 'tsconfig.build.json',
```

Comment:
could you check why this line gives a warning:
```
warn  - Invalid next.config.js options detected: 
  - The root value has an unexpected property, tsconfigPath, which is not in the list of allowed properties (amp, analyticsId, assetPrefix, basePath, cleanDistDir, compiler, compress, crossOrigin, devIndicators, distDir, env, eslint, excludeDefaultMomentLocales, experimental, exportPathMap, generateBuildId, generateEtags, headers, httpAgentOptions, i18n, images, onDemandEntries, optimizeFonts, output, outputFileTracing, pageExtensions, poweredByHeader, productionBrowserSourceMaps, publicRuntimeConfig, reactStrictMode, redirects, rewrites, sassOptions, serverRuntimeConfig, staticPageGenerationTimeout, swcMinify, trailingSlash, typescript, useFileSystemPublicRoutes, webpack).
```

Reply:
[1194 - remove uneccessasry tsconfigPath inside of the config](https://github.com/podkrepi-bg/frontend/pull/1265/commits/56023e4eeaf564c194670bca622ea67e4c0886fa)